nautobot_chatops_ipfabric/worker.py

```diff
@@ -427,6 +432,99 @@ def end_to_end_path(
     return True
 
 
+@subcommand_of("ipfabric")
+def pathlookup(
+    dispatcher, src_ip, dst_ip, src_port, dst_port, protocol
+):  # pylint: disable=too-many-arguments, too-many-locals
+    """Path simulation diagram lookup between source and target IP address."""
+    snapshot_id = get_user_snapshot(dispatcher)
+    sub_cmd = "pathlookup"
+    supported_protocols = ["tcp", "udp", "icmp"]
+    protocols = [(protocol.upper(), protocol) for protocol in supported_protocols]
+
+    dialog_list = [
```

Comment:
isn't this exactly the same dialog_list used in `end_to_end_path` command? should we reuse them?

Reply:
Yes, it is exactly the same now but I believe it will change. I could shift the definition to the top of the module for now anyway if that would be better?

Reply:
if you think this is candidate to change there is no need. Maybe just add a note to give this context for ourselves in the future :)